lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -23,9 +23,14 @@
   name:
     description:
       - name of the database to add or remove
-      - name=all May only be provided if I(state) is C(dump) or C(import).
+      - I(name=all) May only be provided if I(state) is C(dump) or C(import)
+      - name=db_1,db_2 May only be provided if I(state) is C(dump)
+      - name May only be provided if I(state) is C(dump)
+        - db_1
```

Comment:
```suggestion
```

Reply:
We want to keep both the ways to take a dump.
#56721 (comment)

Reply:
Removed